.github/workflows/testing.yml

```diff
@@ -0,0 +1,34 @@
+name: Build and run tests
+
+on:
+  push:
+  pull_request:
```

Comment:
```suggestion
  pull_request:
    branches: [main]
```

Reply:
I agree this should trigger the tests as it stands, but maybe indicating the base branch would help trigger the action?